src/system/PageLockModule/Resources/public/js/PageLock.js

```diff
@@ -1,3 +1,5 @@
+// Copyright Zikula Foundation, licensed MIT.
```

Comment:
This is from https://github.com/zikula/core/wiki/Coding-Standards#javascript-files